lesson2/homework.js

```diff
@@ -0,0 +1,30 @@
+
+
+const addNewTask = function(id, title, description, created, updated, dueDate, statusID, userID) {
```

Comment:
We probably don't want to pass the id as an argument, but maybe we should add a comment stating that they should return the newly added task as an object (including the auto increment ID)